### ISSUE TITLE
1.1.20 Data source without a template does not create RRD ("Extrenal" Data Source Profile)

### ISSUE THREAD

**Issue by tosiara** [Cacti/cacti#954]
Hi,

with 0.8.8f when creating a Data source without a template the RRD is always created:

![image](https://user-images.githubusercontent.com/3600973/30052607-ad05608e-922e-11e7-9591-6e13c0dfbd6c.png)

But now in 1.20 there is no `rrdtool create` command executed:

![image](https://user-images.githubusercontent.com/3600973/30052602-a8a075a6-922e-11e7-8878-5ed29b7fb497.png)
As a result in 1.20 such data sources are ignored by the poller and data is not collected:
```
CMDPHP ERROR: There are no RRA's assigned to local_data_id: 23
```
Is it a bug or now the user must manually create RRD each time a data source is manually created?

**Comment by tosiara** [comment]
Ah, bloody hell, it's a new feature called `The Data Source Profile controls polling interval, the data aggregation, and retention policy for the resulting Data Sources`. And it was set to "External" by default. You have to change it to "system default" if you need the old behavior.

**Comment by tosiara** [comment]
Reopening
If this is a bug rather than feature - then make System default as "default" setting to avoid broken graphs
Thank you

**Comment by cigamit** [comment]
This was fixed in 1.1.22 released last night.
